core/server/data/schema/schema.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
     posts: {
-        id: {type: 'increments', nullable: false, primary: true},
+        id: {type: 'string', nullable: false, primary: true},
```

Comment:
To be honest, when i added to changes i was too lazy adding `maxlength` to each field.
Good that you added a comment. I agree we should define specific length for the id fields. I've read that it can impact memory consumption.